projects/xanpool/project.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Xanpool seems to have some formatting issues. frontmatter doesn't appear properly